__init__.py

```diff
@@ -168,7 +240,7 @@ def reset_face(self, message):
             Sets switches from resting "face" to a registered resting screen.
         """
         time.sleep(1)
-        self.collect_resting_screens()
+        draw_file(self.find_resource('mycroft.fb', 'ui'))
```

Comment:
Reason for the sleep above?

Reply:
Left over from the original mark-2 skill. will remove